docs/tutorials/upgrading-ev3dev-using-shell

```diff
@@ -0,0 +1,24 @@
+---
+title: Upgrading ev3dev Using Shell
+subject: Upgrading
+---
+
+Once you've written the ev3dev image to the SD card, booted, and setup a network 
+connection, you may want to update to the latest kernel.  To do so:
+
+1. Connect to the EV3 using SSH.  (Wireless or USB.)  The default login is 
+username=robot, password=maker.
+
+2. Execute these commands in sequence:
+
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get dist-upgrade
+    sudo reboot
+
+3. Wait for each command to complete before typing the next.  Some steps require 
```

Comment:
It might make more sense for this first sentence (the one about doing them one at a time) to be in the previous step to go with the commands.